pkg/convert/xliff12.go

```diff
@@ -96,25 +96,37 @@ func ToXliff12(translation model.Translation) ([]byte, error) {
 	xlf := xliff12{
 		Version: "1.2",
 		File: xliff12File{
-			SourceLanguage: translation.Language,
 			Body: bodyElement{
 				TransUnits: make([]transUnit, 0, len(translation.Messages)),
 			},
 		},
 	}
 
+	if translation.Original {
+		xlf.File.SourceLanguage = translation.Language
+	} else {
+		xlf.File.TargetLanguage = translation.Language
+	}
```

Comment:
When flow is without an else block, the compiled binary code is identical to your flow `if ... else`. I like this better. Your call.

```suggestion
	xlf.File.TargetLanguage = translation.Language
	if translation.Original {
		xlf.File.SourceLanguage = translation.Language
	}
```

Reply:
I am confused, how is that identical?